app/models/solr_document.rb

```diff
@@ -229,8 +229,12 @@ def alternative_modes
     self[Solrizer.solr_name('alternative_modes')]
   end
 
+  def program_title
+    self[Solrizer.solr_name('program_title')]
+  end
+
   def episode_title
-    self[Solrizer.solr_name('episode_title')]
+    self[Solrizer.solr_name('program_title')]
```

Comment:
@afred don't we need `episode_title` here?

Reply:
@adeelahmad - good catch!